src/brs/http/SubmitNonce.java

```diff
@@ -39,6 +41,8 @@
             .stream()
             .collect(Collectors.toMap(passphrase -> burstCrypto.getBurstAddressFromPassphrase(passphrase).getBurstID().getSignedLongId(), Function.identity()));
     this.allowOtherSoloMiners = propertyService.getBoolean(Props.ALLOW_OTHER_SOLO_MINERS);
+    this.checkPointHeight = propertyService.getInt(propertyService.getBoolean(Props.DEV_TESTNET) ?
```

Comment:
Adding a checkpoint block as an option. The reason of this is for individuals running their wallets with zero configuration as I mentioned before. If someone is running a server version, obviously a database would be configured and this option disabled. But the general distribution should target people with little to no knowledge and should give them a good user experience, not 100% CPU for a day or two.